### DISCUSSION TITLE
SRB TTS not work

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [organicmaps/organicmaps#9647] [Translations]
Text for Serbian TTS exists. In the communication via e-mail, I was told that the text was inserted into the app. When I go to set TTS, there is no Serbian voice, and the Serbian voice is installed.

Somewhere I read your text where it says that the engine is only **eSpeak**.
Please add engines RHVoice, Alfanum (SRB, CRO voices), serbian online google TTS

**Comment by ZeljkoBG** [comment]
To correct after 3 days.

The app does not respond to a change in the TTS engine. Only this morning I saw that the RHVoice Serbian voice is working.

**Reply by biodranik** [reply]
So Serbian TTS works, right? Organic Maps should be restarted after TTS engine changes, see https://github.com/organicmaps/organicmaps/issues/6527

**Reply by ZeljkoBG** [reply]
> Organic Maps should be restarted
**On Android 14, it only works with the Force Stop app.**

I don't want to start a new topic.
Earlier I asked you if the maps could be in Serbian Cyrillic and Latin. You said there are problems... it can.
On the new version, I saw that you added the Serbian Cyrillic language map. Excellent and very important.
My request/question, can the maps be in the Serbian Latin script?
